src/xdsl/diagnostic/lines/xdsl-diagnostic-lines.v6.service.js

```diff
@@ -0,0 +1,64 @@
+angular.module("ovh-api-services").service("OvhApiXdslDiagnosticLinesV6", function ($resource, Poller, OvhApiXdslDiagnosticLines) {
+    "use strict";
+
+    var routes = {
+        base: "/xdsl/:serviceName/lines/:number/diagnostic",
+        cancel: "/xdsl/:serviceName/lines/:number/diagnostic/cancel",
+        run: "/xdsl/:serviceName/lines/:number/diagnostic/run"
+    };
+
+    var interceptor = function (response) {
+        OvhApiXdslDiagnosticLines.resetCache();
+        return response;
+    };
+
+    var diagnostic = $resource(routes.base, {
+        serviceName: "@serviceName",
+        number: "@number"
+    }, {
+        cancelDiagnostic: {
+            url: routes.cancel,
+            method: "POST",
+            isArray: false,
+            interceptor: interceptor
+        }
+    });
+
+    diagnostic.runDiagnostic = function (opts) {
+        var url = routes.run.replace(/\/:(\w*)\//g, function (match, replacement) {
```

Comment:
What ? Could you please add a comment to explain ?

Reply:
Done